### ISSUE TITLE
AVT 1 - React DataTable w/toolbar has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3743]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP IBM Accessibility July 2019 Ruleset

## Detailed Description
Go to the React Datatable Component w/ toolbar
Run DAP and the following issue is found on the copy button (see screenshot):
<img width="1263" alt="Screen Shot 2019-08-15 at 2 38 41 PM" src="https://user-images.githubusercontent.com/21676914/63122515-904f4c80-bf6c-11e9-90b9-3a4ba6896d15.png">

**Comment by joshblack** [comment]
Running DAP on latest gives me the following violations:

![image.png](https://images.zenhubusercontent.com/5618bf72638e718c995dadc6/40b4b955-c3b1-4c2c-8c27-29dfa7681523)

It seems like the interactive element/widget has been updated to have a label:

![image.png](https://images.zenhubusercontent.com/5618bf72638e718c995dadc6/496b8559-6d0c-460b-b60f-2ca64fd7d7c3)
